### PR TITLE
feat(query-service-ts): Victoriametrics support

### PR DESCRIPTION
**Pull request by Tesel**
Add query service for timeseries with VictoriaMetrics Support
Manually tested: Responses from GRPC with victoriametrics answers
* [x] new crate is present in workspace `query-service-ts`
* [x] binary can be queried for datetime range over gRPC
* [x] binary can query VM instance for data (translates gRPC query to promQL)
* [x] `query-service-ts` is added to helm charts
Looks like Quesry router isn't deployed atm, skipping
* [x] `query-service-ts` is added to CD (github Actions)
* [x] *manual* integration tests are written and describe beforementioned points

### CODE REVIEW

**Comment by Kononnable** [comment]
This is finished or should it be marked as a draft? (no description etc.)

**Comment by Tesel** [comment]
> 
> 
> This is finished or should it be marked as a draft? (no description etc.)
Fixed

File: examples/deploy/docker-compose.yml [hunk 1]
```diff
@@ -153,3 +154,29 @@ services:
...
+    build:
+      context: ../..
+      dockerfile: Dockerfile
+      args:
+        - BIN=query-service-ts
+    environment:
+      VICTORIA_QUERY_URL: "http://victoriadb:8428/api/v1/query_range"
+      INPUT_PORT: 50104
+      RUST_LOG: debug
+    links:
```
Comment (frondeus) [inline examples/deploy/docker-compose.yml:172]: > Links are a legacy option. We recommend using networks instead.

Comment (Tesel) [reply examples/deploy/docker-compose.yml]: fixed

**Review by frondeus — DISMISSED** [review]

**Comment by frondeus** [comment]
LGTM but I'd wait with merging and add helm charts

**Review by esavier — CHANGES_REQUESTED** [review]
Looks nice, I see no issues with code so far.
one requested change is to fix Pull Message to conform with angular style, please check other pull requests for clues, also please remember to merge via squash

**Comment by Tesel** [comment]
> 
> 
> Looks nice, I see no issues with code so far.
> one requested change is to fix Pull Message to conform with angular style, please check other pull requests for clues, also please remember to merge via squash

What do you mean with `angular style`?

**Comment by frondeus** [comment]
He meant that in CDL we try to follow a similar commit format like in https://gist.github.com/brianclements/841ea7bffdb01346392c

**Review by esavier — APPROVED** [review]

**Review by frondeus — APPROVED** [review]

**Review by esavier — APPROVED** [review]

**Pull request merged**